docs/changelog/2024/2412.md

```diff
@@ -0,0 +1,127 @@
+    - Transitions renamed.
+- The following SIDs have been renamed:
+    - `ELSAB 2P` becomes the `ONODA 1P`: Initial departure routing slightly changed. `PUHOI` transition added.
+- The following RNP has been amended: `RNP X RWY 23L (AR)`
+    - Final arcing turn slightly amended. Missed approach path amended.
+
+#### NZCH: Christchurch
+
+- The following STARs have been amended and resequenced: All `BELEE` STARs, `UPVES 2D`.
+    - WPT `BELEE` altitude restrictions removed.
+
+#### NZHN: Hamilton
+
+- Hamilton Delivery position (HN_DEL) has been removed.
+- The following STARs have been amended and resequenced: `YOGIT 3B`, `YOGIT 3D`.
+    - `LAKES` transition replaced by `ORIDI`.
+- The following STAR has been renamed: `LAKES 3C` becomes the `ORIDI 1C`.
+- VFR procedures added.
+
+#### NZHK: Hokitika
+
+- The following SIDs have been added: `DOPAX 1R`, `OSOLO 1R`, `DOPAX 1S`, `OSOLO 1S`.
+
+#### NZKK: Kerikeri
+
+- The following STARs have been added: `SAKIM 1A`, `URGAR 1A`, `DUKEG 1B`, `URGAR 1B`, `SAKIM 1B`.
+- The following SIDs have been added: `UPGAM 1P`, `OBKOP 1P`, `UPGAM 1Q`, `OBKOP 1R`, `OBKOP 1Q`.
+- The following SIDs have been removed: `UKAPA 2A`, `UKAPA 1B`.
+- The following RNPs have been amended: `RNP RWY 15`, `RNP RWY 33`.
+    - All waypoints replaced. RNP is now a T-bar, with three IAFs.
+
+#### NZNR: Napier
+
+- CTR/D now separated into three sectors: `West`, `Instrument` and `East`.
+- VFR procedures added.
+
+#### NZNP: New Plymouth
+
+- The following SID has been amended: `NP 1`.
+    - WPT `KARRL` replaced by `IGBAT`.
+    - Procedure not resequenced.
+
+#### NZWN: Wellington
+
+- `South East` sector renamed `Turakirae`.
+
+#### NZWP: Whenuapai 
+
+- The following SID has been amended and resequenced: `BAYES 2R`.
+    - `LAKES` transition replaced by `ORIDI`.
+- The following SIDs have been added: `NZWP 1A`, `NZWP 1B`.
+
+#### NZWB: Woodbourne
+
+- The following SIDs have been amended and resequenced: `VINES 5`, `VALLEY 4`.
+    - `VINES 5` departure routing amended.
+- The following SIDs have been added: `BEAVER 1`.
+
+-----
+
+### Standard Routes
+
+- The following STD RTEs have been amended and resequenced: `AANP2`, `AAPM3`, `AAPP1`, `AAWN2`, `CHWN7`, `WRWN1`, `WPOH1`, `WPWN2`.
+- The following STD RTEs have been added: `ARHN1`
+- All Dunedin STD RTEs have had their remarks amended to clarify the `FL260` or higher requirement, with the `FL250` hold-down now reflected in DN TWR SOPs.
+- Further standardised remark values.
+- Routes have been updated in the Standard Route Tool and [public route repository](https://github.com/vatnz-dev/std-rte-public){ target=blank }.
+
+### VRPs for LittleNavMap
+
+The VRPs for LittleNavMap package has been updated, and [may be downloaded here](https://github.com/vatnz-dev/vrpsForLittleNavMap/releases/tag/2412){ target=blank }.
+
+### Other Dataset Changes
+
+- The Hamilton Delivery position (HN_DEL) has been removed.
```

Comment:
This one is referenced both here and in the data changes for NZHN. Guessing we should probably go with one or the other?

Reply:
I think we should leave it in the Hamilton section - as it's a AIP-directed change, as opposed to a VATNZ-directed change.